### ISSUE TITLE
The Color Contrast ratio of the Column header text is (1.008:1) which is less than required 4.5:1 with respect to it's background: A11y_Testing, Interactive and ARM tools_Polyglot Notebook variable section_AI4D

### ISSUE THREAD

**Issue by shindepra** [dotnet/interactive#4005]
### GitHub Tags:
#A11yTCS;#A11yMAS;#Win11;#DesktopApp;#A11ySev2;#BM_Testing-InteractiveandARMtools_Win32_July2025;#Testing, Interactive and ARM tools;#WCAG1.4.3;#Contrast(minimum);#Benchmark;#Win32;#FTP;
 
### Environment Details:
Application Name: Testing, Interactive and ARM tools
Visual Studio Code : polyglot Notebook Extension
OS Details:
Microsoft Windows 11 Enterprise Version 21H1 (OS build 26100.4652)
 
### Repro Steps:
1.  launch "Visual Studio Code" application.
2.Install .NET 6.0 SDK using https://dotnet.microsoft.com/en-us/download/dotnet/thank-you/sdk-6.0.402-windows-x64-installer.
3.Install Polyglot Notebook extension. 
4. launch Visual studio code insider and Ctrl+Alt+Shift+N will invoke "Polyglot Notebook: Create new blank notebook" command used above.
5. Enter any code and run the code.
6. Tab till values and press enter.
7.In polyglot notebooks values section.
8. Tab till Column header in grid section and observe that the Color Contrast ratio of the Column header text is greater than or equal to required 4.5:1 or not.
 
### Actual Result:
The Color Contrast ratio of the Column header text is which is (1.008:1) which is less than required 4.5:1 with respect to its background and not clearly visible.
 
### Expected Result:
The Color Contrast ratio of the Column header text should be greater than or equal to required 4.5:1 with respect to its background and should be clearly visible.
 
### User Impact:
low vision and visually impaired users will face difficulty in viewing the text if the color contrast ratio of text is less than required 4.5:1 with respect to its background
 
### Attachments for reference:

![Image](https://github.com/user-attachments/assets/a1939405-8c75-4f26-b379-9bbbd04b3aee)
